modules/prodigal/meta.yml

```diff
@@ -1,32 +1,27 @@
 name: prodigal
-## TODO nf-core: Add a description of the module and list keywords
-description: write your description here
+description: Prodigal (Prokaryotic Dynamic Programming Genefinding Algorithm) is a microbial (bacterial and archaeal) gene finding program
```

Comment:
Might be cleaner to keep this PR clean with only version changes

Reply:
BTW, after writing the comment it sounds really very @drpatelh -ish, am I wrong? :smile:

Reply:
Nope. My bad. Should have done it in a separate PR. You are right. Trying to do 1000 things so squeezed it in here 😬

Reply:
These came up when I was testing `nf-core lint` which is why I fixed them.